### PR TITLE
test: enhance e2e test to setup the election as a user

### PR DESCRIPTION
**Pull request by beagleknight**
I added the setup election process to the e2e test as a preparation to add a complete test for the `election_guard` voting scheme name.

### CODE REVIEW

File: decidim-bulletin_board-ruby/lib/decidim/bulletin_board/authority/create_election.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def call
 
         private
 
-        attr_reader :election_data, :election_id
+        attr_reader :election_data, :election_id, :election_voting_scheme_name
```
Comment (leio10) [inline decidim-bulletin_board-ruby/lib/decidim/bulletin_board/authority/create_election.rb:45]: Mmm, is this needed?

**Review by leio10 — APPROVED** [review]
Great work! 👍

**Review by orlera — APPROVED** [review]
Nice!

File: decidim-bulletin_board-ruby/lib/decidim/bulletin_board/authority/create_election.rb (hunk 1 above)
Comment (beagleknight) [reply decidim-bulletin_board-ruby/lib/decidim/bulletin_board/authority/create_election.rb:45]: Nope, that's old stuff!

**Pull request merged**